client/me/purchases/remove-purchase/index.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Dialog from 'components/dialog';
+import { getPurchase, isDataLoading } from '../utils';
+import { getName, isExpiring } from 'lib/purchases';
+
+const RemovePurchase = React.createClass( {
+	propTypes: {
+		selectedPurchase: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired
+	},
+
+	getInitialState() {
+		return {
+			isDialogOpen: false
```

Comment:
Shouldn't we name this property `showDialog` as we did in other components? That's what the [readme file](https://github.com/Automattic/calypso-pre-oss/tree/master/client/components/dialog) suggests in its examples.